src/zoid/payment-fields/component.jsx

```diff
@@ -122,6 +122,11 @@ export function getPaymentFieldsComponent() : PaymentFieldsComponent {
                     required: false
                 },
 
+                onContinue: {
```

Comment:
What is the purpose of this callback? Wondering if it is the same as `onComplete` that was created for flows where we capture or authorize on our servers.  `onComplete` would be great for Apple Pay, for example.

Reply:
..yh..

Reply:
@mnicpt We are trying to render the payment-fields inline for APMs, for now lets say `eps`. If you see in the screenshot after onclick of eps from vertical stack `PaymentFields` iframe will be rendered inline and it has a Continue button. This `onContinue` callback is been used for this button.
<img width="642" alt="image" src="https://user-images.githubusercontent.com/22331155/183573786-7548e7e1-db8b-437d-b686-721e6dbeae88.png">